src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,46 @@
+"use client";
+
+import styles from "./Sidebar.module.sass";
+
+export default function Sidebar({
+  showSidebar,
+  toggleSidebar,
+}: {
+  showSidebar: boolean;
+  toggleSidebar: () => void;
+}) {
+  return (
+    <aside
+      className={`${styles.sidebar} ${showSidebar ? styles.active : null}`}
+    >
+      <nav>
+        <ul>
+          <button onClick={toggleSidebar} className={styles.closeButton}>
+            <svg width="14" height="15" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It might be better to create a component for the SVG instead of directly adding them here, same with the others.

Reply:
created a separate SvgComponents.tsx file to house all svg exports